README.md

```diff
@@ -1 +1,26 @@
-# Enumerable-ruby-proj
\ No newline at end of file
+# Enumerable-ruby-proj
+
+Created Enumerable to better understand how module classes work in ruby. 
+
+# Getting started
+- ruby install
+- clone repo
+- run on your local machine
```

Comment:
- [ ] I would recommend including the steps to clone and run the project locally to make the README professional.

```
git clone https://github.com/SirriRyisa/Enumerable-ruby-proj.git
cd Enumerable-ruby-proj
ruby my_list.rb
```

Reply:
Let's address this issue here as advised by previous reviewer. The getting started section of the readme file should have clear instructions for running the project on the user's local machine.